build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 // please update RELEASE_NOTES.md when you update this version
-def buildVersion = "2.1.0"
+def buildVersion = "2.1.1"
```

Comment:
[nit] this feels like a minor (add functionality in a backward compatible manner) rather than a patch (backward compatible bug fixes) ?

Reply:
Yeah you're right, functionality is a minor version bump

Reply:
Updated